idea/testData/inspectionsLocal/moveVariableDeclarationIntoWhen/multiLinePropertywithSingleLineInitializer2.kt

```diff
@@ -0,0 +1,10 @@
+fun test() = 42
+
+fun foo() {
+    val <caret>a =
```

Comment:
I think this is incorrect behavior. Inspection should be available in cases:
* this is one-line property
* this is property with one-line initializer and action == INLINE